pyspades/server.py

```diff
@@ -91,19 +91,19 @@ def __init__(self, *arg, **kw):
         self.entities = []
         self.players = MultikeyDict()
         self.player_ids = IDPool()
-        self.spectator_team = self.team_class(-1, self.spectator_name,
+        self.team_spectator = self.team_class(-1, self.spectator_name,
                                               (0, 0, 0), True, self)
-        self.blue_team = self.team_class(0, self.team1_name, self.team1_color,
-                                         False, self)
-        self.green_team = self.team_class(1, self.team2_name, self.team2_color,
-                                          False, self)
+        self.team_1 = self.team_class(0, self.team1_name, self.team1_color,
+                                      False, self)
+        self.team_2 = self.team_class(1, self.team2_name, self.team2_color,
+                                      False, self)
         self.teams = {
             -1: self.spectator_team,
-            0: self.blue_team,
-            1: self.green_team
+            0: self.team_1,
+            1: self.team_2
         }
-        self.blue_team.other = self.green_team
-        self.green_team.other = self.blue_team
+        self.team_1.other = self.team_2
```

Comment:
Not that we should remove it now, but is there any particular reason this exists? Seems unnecessary

Reply:
well, how would you programmatically access the other team otherwise? You could write an if elif thing every time, but I think this is a neat hack to shorten that.